manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -1699,7 +1699,7 @@ data:
                 - name: ISTIO_META_APP_CONTAINERS
                   value: ""
                 - name: ISTIO_META_CLUSTER_ID
-                  value: "{{ valueOrDefault .Values.global.multiCluster.clusterName `Kubernetes` }}"
+                  value: "{{ valueOrDefault .Values.global.multiCluster.clusterName .ClusterID }}"
```

Comment:
should .ClusterID be first actually? Or only with no default?

TBH not sure I full understand how these are set :-)